incubator/hnc/internal/validators/object_test.go

```diff
@@ -308,6 +309,20 @@ func TestUserChanges(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "Deny creation of object with invalid select annotation",
+		fail: true,
+		inst: &unstructured.Unstructured{
+			Object: map[string]interface{}{
+				"apiVersion": "v1",
+				"kind":       "Pod",
+				"metadata": map[string]interface{}{
+					"annotations": map[string]interface{}{
+						api.AnnotationSelector: invalidSelStr,
```

Comment:
I feel like the string "a b c" would also be a good illegal labelSelector

Reply:
Changed to `$foo`